mithril-client/src/services/snapshot.rs

```diff
@@ -168,6 +169,19 @@ impl MithrilClientSnapshotService {
             res = future => res,
         }
     }
+
+    async fn expand_eventual_snapshot_alias(&self, snapshot_id: String) -> StdResult<String> {
+        if snapshot_id == "latest" {
```

Comment:
Ideally we should be case insensitive, to do that you can just lowercase the `snapshot_id`.